### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by briantting**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-# ![Logo](chrome/app/theme/chromium/product_logo_64.png) Chromium
+# ![Logo](chrome/app/theme/chromium/product_logo_64.png) Chromium [![Build Matrix](https://img.shields.io/badge/-Build%20Matrix-blueviolet)](https://github.com/youtube/cobalt/blob/main/BUILD_STATUS.md)
+
+[![android](https://github.com/youtube/cobalt/actions/workflows/android.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/android.yaml?query=event%3Apush+branch%3Amain)
+[![linux](https://github.com/youtube/cobalt/actions/workflows/linux.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/linux.yaml?query=event%3Apush+branch%3Amain)
+[![win32](https://github.com/youtube/cobalt/actions/workflows/win32.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/win32.yaml?query=event%3Apush+branch%3Amain)
+[![xb1](https://github.com/youtube/cobalt/actions/workflows/xb1.yaml/badge.svg?branch=main&event=push)](https://github.com/youtube/cobalt/actions/workflows/xb1.yaml?query=event%3Apush+branch%3Amain)
```
Comment (oxve) [inline README.md:6]: This is fake news, win32 and xb1 are not built :)

**Review by oxve — APPROVED** [review]
Just remove win32 and xb1, otherwise good

File: README.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-# ![Logo](chrome/app/theme/chromium/product_logo_64.png) Chromium
```
Comment (dahlstrom-g) [inline README.md:1]: Can we avoid modifying upstream lines?

**Review by isarkis — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (oxve) [reply README.md:1]: This whole files needs to be replaced.

**Review by kaidokert — APPROVED** [review]

**Pull request merged**